string sample_type
---
bool sample_result